brands/amenity/bank.json

```
@@ -873,6 +884,26 @@
       "name": "Banca March"
     }
   },
+  "amenity/bank|Banca Mediolanum": {
+    "countryCodes": ["it"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "Banca Mediolanum",
+      "brand:wikidata": "Q3633712",
+      "brand:wikipedia": "en:Banca Mediolanum",
+      "name": "Banca Mediolanum"
+    }
+  },
+  "amenity/bank|Banca Popolare di Bari": {
+    "countryCodes": ["it"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "Banca Popolare di Bari",
+      "brand:wikidata": "Q3633730",
+      "brand:wikipedia": "en:Banca Popolare di Bari",
+      "name": "Banca Popolare di Bari",
```

```suggestion
      "name": "Banca Popolare di Bari"
```

This last line should not have a comma